components/ClaimModal/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './ClaimModal';
```

Comment:
curious why the need for a directory and index file for each component. Can't we just create the component files and use them directly?

Reply:
For this case it may look unreasonable right, but if you have multiple files related to the same component you need a seperate folder. 
But this way you will have to use double name while importing `import { MyComponent } from'/components/MyComponent/MyComponent';`

Solution to this is naming the component file to `index.tsx` but this is not a good experience working in IDE,
a) Multiple index.tsx tabs in editor maybe confusing (https://twitter.com/trkaplan/status/1009352393748774912?s=20&t=m3XTBI5BL4FrG_a3_LVW0Q)
b) I use Command Panel to open files in project so it maybe confusing seeing a list of index.tsx (they have comp. name in the path but It is better to see just component's name)

Reply:
Yeah, but you can export as default, then you don't need `import { MyComponent } from...`, you just import as `import MyComponent from...`.

But also, it's ok to use in those rare circumstances that you really need multiple files, but it's overkill for 90% of the time when you don't need it.